packages/webgpu/src/Canvas.tsx

```diff
@@ -11,50 +11,19 @@ import {
 } from "react";
 import type { RefObject } from "react";
```

Comment:
shouldn't this PR only add `src/Canvas.web.tsx`,  and `src/main.web.ts` and that's it sorry if I am missing something.